### ISSUE TITLE
Improved Fly-over

### ISSUE THREAD

**Issue by tedbarnett** [TimeWalkOrg/Manhattan#8]
Our current fly-over (pressing "G" to generate while playing a level) needs the following improvements:

- Slower start (i.e. wait 3-5 seconds before starting movement, slower swoop down to the water)
- Should work for 1776 level (which will soon be based on 1776_CE)
- Should spend more time slowly flying over the area below Wall Street so that we can show both 1660 and 1776 buildings
- Nice-to-have: can somehow show night and day
- Nice-to-have: can somehow show off seasons changing
